fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body class="container">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
Good job using an hx tag here :clap:  but I'll suggest you to put these 2 lines inside a **header tag**.

**Note:** The **header tag** element represents a container for introductory content or a set of navigational links.

Reply:
Outstanding work @ForHemer 🔥👀🔥

Reply:
Excellent 🧐